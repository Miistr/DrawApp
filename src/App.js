import React from "react";
import CanvasDraw from "./components/CanvasDraw/CanvasDraw";
import "./App.css";

const App = () => (
  <div className="app-container">
    <CanvasDraw />
  </div>
);

export default App;
