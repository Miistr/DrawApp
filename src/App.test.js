import React from "react";
import App from "./App";
import renderer from "react-test-renderer";

describe("Given <App/> component", () => {
  let component;
  beforeEach(() => {
    component = renderer.create(<App />);
  });
  test("should match the snapshot", () => {
    expect(component.toJSON()).toMatchSnapshot();
  });
});
