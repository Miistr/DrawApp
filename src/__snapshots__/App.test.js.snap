// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Given <App/> component should match the snapshot 1`] = `
<div
  className="app-container"
>
  <div
    className="input-block"
  >
    <input
      accept=".txt"
      onChange={[Function]}
      type="file"
    />
  </div>
</div>
`;
